services/housetables/src/main/java/com/linkedin/openhouse/housetables/repository/impl/jdbc/UserTableHtsJdbcRepository.java

```diff
@@ -42,19 +42,20 @@ Optional<UserTableRow> findByDatabaseIdIgnoreCaseAndTableIdIgnoreCase(
   @Query("SELECT DISTINCT databaseId FROM UserTableRow")
   Iterable<String> findAllDistinctDatabaseIds();
 
-  Iterable<UserTableRow> findAllByDatabaseIdIgnoreCase(String databaseId);
+  Iterable<UserTableRow> findAllByDatabaseIdIgnoreCaseAndDeletedIsFalse(String databaseId);
 
-  Iterable<UserTableRow> findAllByDatabaseIdAndTableIdLikeAllIgnoreCase(
+  Iterable<UserTableRow> findAllByDatabaseIdAndTableIdLikeAllIgnoreCaseAndDeletedIsFalse(
```

Comment:
So we will get all the tables by default if they are soft deleted?

Reply:
We might need to add query for these methods to exclude soft deleted ones.

Reply:
For better readability it would be good to add the Query directly although Spring data JPA parses the method name and generates query.